projects/packages/my-jetpack/_inc/components/welcome-flow/index.tsx

```diff
@@ -47,35 +47,24 @@ const WelcomeFlow: FC< Props > = ( {
 	const [ prevStep, setPrevStep ] = useState( '' );
 
 	const currentStep = useMemo( () => {
-		if ( ! siteIsRegistered || welcomeFlowExperiment.isLoading ) {
+		if ( ! siteIsRegistered ) {
 			return 'connection';
 		} else if ( ! isProcessingEvaluation ) {
 			if ( ! recommendedModules && ! isJetpackUserNew() ) {
 				// If user is not new but doesn't have recommendations, we skip evaluation
 				// If user has recommendations, it means they redo the evaluation
 				return null;
 			}
-			// For the "treatment" experiment we immediately jump to the 'evaluation-processing' step if
-			// there are no `recommendedModules` loaded yet.
-			if (
-				'treatment' === welcomeFlowExperiment.variation &&
-				! recommendedModules &&
-				isJetpackUserNew()
-			) {
+
+			if ( ! recommendedModules && isJetpackUserNew() ) {
 				return 'evaluation-processing';
 			}
```

Comment:
I think in this case, we need to remove the condition altogether, as it was only applicable for 'treatment'.

Currently, the flow gets stuck on the processing step, as I'm testing it.

Reply:
I removed this piece of code in the other PR, but I'll remove it here as well since it affects the behavior of the connection.